src/main/java/com/spikes2212/command/drivetrains/TankDrivetrain.java

```diff
@@ -38,6 +39,14 @@ public TankDrivetrain(MotorController leftController, MotorController rightContr
         this(getClassName(DEFAULT_NAMESPACE_NAME), leftController, rightController);
     }
 
+    public TankDrivetrain(MotorController leftMaster, MotorController leftSlave, MotorController rightMaster, MotorController rightSlave) {
```

Comment:
put this below the consturctor which takes (String, MotorController, MotorController, MotorController, MotorController)